serial_communication/onboard/onboard/ConstJoint.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CONSTJOINT_H_
+#define CONSTJOINT_H_
+
+struct ConstJoint {
```

Comment:
I think the name ConstJoint is unclear as to what this struct represents. It sounds like a joint that will never move, not the constants associated with the joint. I think something like JointParams or JointAttributes would make more sense.

Additionally, there is a lot in this struct, which should probably split into multiple substructures